module/registry/util/src/main/java/org/openbase/bco/registry/print/BCOQueryLabelPrinter.kt

```diff
@@ -0,0 +1,248 @@
+package org.openbase.bco.registry.print
+
+import org.openbase.jps.core.JPService
+import org.openbase.bco.registry.remote.Registries
+import org.openbase.bco.registry.remote.login.BCOLogin
+import org.openbase.jul.exception.CouldNotPerformException
+import org.openbase.jul.exception.NotAvailableException
+import org.openbase.jul.exception.printer.ExceptionPrinter
+import org.openbase.jul.extension.type.processing.LabelProcessor
+import org.openbase.jul.extension.type.processing.ScopeProcessor
+import org.openbase.jul.processing.StringProcessor
+import org.openbase.jul.processing.StringProcessor.Alignment.RIGHT
+import java.lang.InterruptedException
+import org.openbase.type.domotic.unit.UnitConfigType.UnitConfig
+import org.openbase.type.domotic.state.EnablingStateType.EnablingState.State.ENABLED
+import java.lang.Exception
+import kotlin.system.exitProcess
+
+/*
+ * #%L
+ * BCO Registry Utility
+ * %%
+ * Copyright (C) 2014 - 2021 openbase.org
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */ /**
+ * @author [Divine Threepwood](mailto:divine@openbase.org)
+ */
+object BCOQueryLabelPrinter {
+    const val APP_NAME = "bco-id-to-label"
```

Comment:
This name does not match the generated binay `bco-query-label` as defined in build.gradle.kts